excellent/functions/builtin.go

```diff
@@ -37,7 +37,7 @@ func init() {
 		"char":              OneNumberFunction(Char),
 		"code":              OneTextFunction(Code),
 		"split":             TwoTextFunction(Split),
-		"join":              ArgCountCheck(2, 2, Join),
+		"join":              TwoArgFunction(Join),
```

Comment:
unrelated change.. just remembered there's a better wrapper for a function that takes a fixed number of args